setup.py

```diff
@@ -14,6 +14,14 @@
         "psycopg2==2.7.5",
         "inflection==0.3.1"
     ],
```

Comment:
I think this is why the install is failing - you are installing these pinned versions. We use psycopg 2.9.3, so this an old and maybe less compatible version

But I wouldn't pin versions in setup. Maybe do `>=2.7.5` ? or remove the version pin and try again